### PR TITLE
Explain Rancher access scope in external auth docs

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses this issue: https://github.com/rancher/docs/issues/1683

### CODE REVIEW

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md [hunk 1]
```diff
@@ -40,6 +40,28 @@ Rancher relies on users and groups to determine who is allowed to log in to Ranc
 
 For more information, see [Users and Groups]({{< baseurl >}}/rancher/v2.x/en/admin-settings/authentication/user-groups/)
 
+## Scope of Rancher Authorization
+
+After you configure Rancher to allow sign on using an external authentication service, you should configure who should be allowed to log in and use Rancher. The following options are available:
+
+| Access Level | Description |
+|----------------------------------------------|-------------|
+| Allow any valid Users                   | _Any_ user in the authorization service can access Rancher. We generally discourage use of this setting! |
```
Comment (deniseschannon) [inline content/rancher/v2.x/en/admin-settings/authentication/_index.md]: @mrajashree Is this setting usable for any of the SAML based providers? Or for openLDAP? The old 1.6 docs only said it was available for GitHub/AD

**Comment by deniseschannon** [comment]
Are the caveats in this doc still relevant in 2.3? 

https://rancher.com/docs/rancher/v1.6/en/configuration/access-control/#site-access

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md (hunk 1 above)
Comment (mrajashree) [reply content/rancher/v2.x/en/admin-settings/authentication/_index.md]: @deniseschannon yes all settings are usable. The saml based providers only have a problem with lookup. But once the users are added to either `Authorized Users and Organizations` list or cluster/project members list these settings work for saml the same way

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md [hunk 2]
```diff
@@ -40,6 +40,28 @@ Rancher relies on users and groups to determine who is allowed to log in to Ranc
...
+| Allow members of Clusters, Projects, plus Authorized Users and Organizations | Any user in the authorization service and any group added as a **Cluster Member** or **Project Member** can log in to Rancher. Additionally, any user in the authentication service or group you add to the **Authorized Users and Organizations** list may log in to Rancher. |
+| Restrict access to only Authorized Users and Organizations | Only users in the authentication service or groups added to the Authorized Users and Organizations can log in to Rancher. |
+
+To set the Rancher access level for users in the authorization service, follow these steps:
+
+1. From the **Global** view, click **Security > Authentication.**
+
+1. Use the **Site Access** options to configure the scope of user authorization. The table above explains the access level for each option.
+
+1. Optional: If you choose an option other than **Allow any valid Users,** you can add users to the list of authorized users and organizations by searching for them in the text field that appears.
```
Comment (mrajashree) [inline content/rancher/v2.x/en/admin-settings/authentication/_index.md]: we can add the SAML caveat here, that this lookup won't work for SAML based providers

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md (hunk 1 above)
Comment (prachidamle) [reply content/rancher/v2.x/en/admin-settings/authentication/_index.md]: This is available for all auth providers in 2.x

File: content/rancher/v2.x/en/admin-settings/authentication/_index.md (hunk 2 above)
Comment (deniseschannon) [reply content/rancher/v2.x/en/admin-settings/authentication/_index.md]: @catherineluse we should use the shortcode for the saml caveat. like this: 

https://github.com/rancher/docs/blame/master/content/rancher/v2.x/en/admin-settings/authentication/keycloak/_index.md#L61

Comment (catherineluse) [reply content/rancher/v2.x/en/admin-settings/authentication/_index.md]: added shortcode

**Pull request merged**